File: Sem4/MPP/cs/lab8cs/Model/Spectacol.cs
using System.Runtime.InteropServices.JavaScript;

namespace lab8cs.Model;

public class Spectacol:Entity<long>
{
    public Spectacol(string artist, DateTime date, string location, int availableSeats, int soldSeats)
    {
        Artist=artist;
        Date=date;
        Location=location;
        AvailableSeats=availableSeats;
        SoldSeats=soldSeats;
    }
    public string Artist { get; set; }
    public DateTime Date { get; set; }
    public string Location { get; set; }
    public int AvailableSeats { get; set; }
    public int SoldSeats { get; set; }

    public override string ToString()
    {
        return Artist + " : " + Location + " : " + Date.ToShortDateString()+":"+AvailableSeats+":"+SoldSeats;
    }
}

File: Sem4/MPP/java/lab6/Domain/src/main/java/ubb/scs/mpp/model/Spectacol.java
package ubb.scs.mpp.model;
import java.time.LocalDateTime;

public class Spectacol extends Entity<Long> {
    private String artist;
    private LocalDateTime date;
    private String location;
    private Integer available_seats;
    private Integer sold_seats;

    public Spectacol(String artist, LocalDateTime date, String location, Integer available_seats, Integer sold_seats) {
        this.artist = artist;
        this.date = date;
        this.location = location;
        this.available_seats = available_seats;
        this.sold_seats = sold_seats;
    }

    public String getArtist() {
        return artist;
    }

    public void setArtist(String artist) {
        this.artist = artist;
    }

    public LocalDateTime getDate() {
        return date;
    }

    public void setDate(LocalDateTime date) {
        this.date = date;
    }

    public String getLocation() {
        return location;
    }

    public void setLocation(String location) {
        this.location = location;
    }

    public Integer getAvailable_seats() {
        return available_seats;
    }

    public void setAvailable_seats(Integer available_seats) {
        this.available_seats = available_seats;
    }

    public Integer getSold_seats() {
        return sold_seats;
    }

    public void setSold_seats(Integer sold_seats) {
        this.sold_seats = sold_seats;
    }

    @Override
    public String toString() {
        return "Spectacol [artist=" + artist + ", date=" + date + ", location=" + location+ ", available_seats=" + available_seats+ ", sold_seats=" + sold_seats + "]";
    }
}
